rust/src/sip/parser.rs

```diff
@@ -78,7 +78,7 @@ fn is_request_uri_char(b: u8) -> bool {
 
 #[inline]
 fn is_version_char(b: u8) -> bool {
-    is_alphanumeric(b) || b"./".contains(&b)
+    b"SIP/2.0".contains(&b)
```

Comment:
I think we rather want the caller of `is_version_char` begin by `tag("SIP/")`, what do you think ?

Reply:
I agree with you, it's much better tagging `SIP/` string first and then taking the chars until `\r\n` is found.